DISCOVER/07_code_of_conduct.md

```diff
@@ -69,19 +69,21 @@ Make sure that **everyone involved in your conference/event** is aware that the
 **Why You Need a Code of Conduct**
 - [Why to have a CoC and how to make one](http://incisive.nu/2014/codes-of-conduct/)
 - [Why conferences need a code of conduct](https://jacobian.org/writing/codes-of-conduct/)
-- [You literally cannot pay me to speak without a Code of Conduct](http://rachelnabors.com/2015/09/01/code-of-conduct/)
+- The importance of having a code of conduct cannot be overstated. [^code-of-conduct]
+[^code-of-conduct]: Rachel Nabors, "You Literally Cannot Pay Me to Speak Without a Code of Conduct," published September 1, 2015. Available at: [Rachel Nabors' Blog](http://rachelnabors.com/2015/09/01/code-of-conduct/) (currently unavailable).
```

Comment:
Once again having the footnote in a list causes a rendering issue. 

This PR:

![Screenshot 2025-01-15 at 6 22 29 AM](https://github.com/user-attachments/assets/41d982f0-bf4e-44a4-ae63-4e92086e7772)

Production version

![Screenshot 2025-01-15 at 6 22 52 AM](https://github.com/user-attachments/assets/5b855f86-8748-46cd-8740-55ab045ff3ed)

To be honest though, I suggest we remove this line entirely. It is a list of further reading but the link is no longer published, thus making it impossible to read.